hoover/search/es.py

```diff
@@ -1,17 +1,39 @@
+import json
+from contextlib import contextmanager
 from django.conf import settings
 from elasticsearch import Elasticsearch
 from elasticsearch.client.utils import _make_path
 from elasticsearch.exceptions import NotFoundError, RequestError, ConnectionError
 
-es = Elasticsearch(settings.HOOVER_ELASTICSEARCH_URL)
 DOCTYPE = 'doc'
 
 class SearchError(Exception):
     def __init__(self, reason):
         self.reason = reason
 
+@contextmanager
+def elasticsearch():
+    try:
+        yield Elasticsearch(settings.HOOVER_ELASTICSEARCH_URL)
+    except Exception: raise
```

Comment:
I don't think we need this line. It might actually mask the `except` clauses below.